app/jenkins-for-jira-ui/src/components/ConnectionPanel/NotConnectedState.tsx

```diff
@@ -111,27 +111,28 @@ const NotConnectedState = ({
 							<Spinner size='large' />
 						</div>
 					) : (
-						<>
-							<ConnectionPanelContent
+						<> {isPending
```

Comment:
I feel that this was clearer in it's original state. I think this component needs a big refactor, but having smaller ternary conditions in each prop is clearer than having two separate components. Up to you whether to revert it though

Reply:
I did try not to split to two separate component in the beginning. But the small ternary conditions turns out to be not very readable while im trying to figure out what exactly the component should render. It becomes even harder when added more "can be undefine parameter" to the props.